docs/api-reference/cli.md

```diff
@@ -106,6 +106,22 @@ PORT=4000 npx next start
 
 > Note: `PORT` can not be set in `.env` as booting up the HTTP server happens before any other code is initialized.
 
+### Working with Load-balancer Timeouts
+
+When deploying Next.js behind an upstream proxy (e.g. a load-balancer like AWS ELB/ALB) it's important to configure Next's underlying http-server with [keep-alive timeouts](https://nodejs.org/api/http.html#http_server_keepalivetimeout) that are _smaller_ than the upstream proxies' timeouts. Otherwise, once a keep-alive timeout is reached for a given TCP connection, Node.js will immediately terminate that connection without notifying the upstream proxies. This results in the upstream proxies throwing intermittent 5xx errors whenever they attempt to reuse a connection that Node.js has already terminated.
```

Comment:
All mentions of "upstream" in this description should be "downstream".

- Next.js is "upstream"
- Proxy is "downstream"

Reply:
FWIW, I _kinda_ disagree with this suggestion. If the request comes into the LB initially, then Next.js is "downstream" from it.

Reply:
yea, it's a matter of perspective. i'm okay going either way — if you don't feel too strongly about it, I'm just going to go w/ @styfle suggestion :)